generator/src/generator/namespace/mod.rs

```diff
@@ -1518,6 +1518,8 @@ impl<'ns, 'c> NamespaceGenerator<'ns, 'c> {
                     derives.ord = false;
```

Comment:
Line 1514 talks about `RawFdContainer`. That comment might need some updating. :-)

Reply:
Fixed!